agent/engine/docker_task_engine.go

```diff
@@ -573,6 +575,7 @@ func (engine *DockerTaskEngine) deleteTask(task *apitask.Task) {
 		if taskENI.IsStandardENI() {
 			seelog.Debugf("Task engine [%s]: removing eni %s from agent state",
 				task.Arn, taskENI.ID)
+			engine.removeENIAttachment(taskENI.MacAddress)
```

Comment:
rename to `removeENIAttachmentFromDataClient` ?

Reply:
Renamed to removeENIAttachmentFromData.